manifests/b/Bloodstrike/pc/1.1/Bloodstrike.pc.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Bloodstrike.pc
+PackageVersion: "1.1"
+PackageLocale: en-US
+Publisher: Netease
+PackageName: Blood Strike
+License: Netease
+Copyright: (C) Netease 保留所有权利。
+ShortDescription: Blood Strike permite que você junte-se a amigos rapidamente nos modos em equipe. Encontrar seus colegas, adicioná-los ao esquadrão e iniciar uma partida é superfácil. Blood Strike foi criado para dispositivos mobile.

```

Comment:
Please update this so that the text's language matches the locale file's language.